spectrum/display.lua

```diff
@@ -141,8 +141,11 @@ function Display:_drawActors(drawnActors, queryable, alpha)
          tempColor.a = tempColor.a * alpha
 
          --- @diagnostic disable-next-line
-         local ax, ay = actor.position:decompose()
-         self:putDrawable(x + ax, y + ay, drawable, tempColor)
+         local position = actor:getPosition()
+         if position then
```

Comment:
Could you just query for position? And there's another check missing in `putLevel`.